File: .snippets/code/ethers-tx-local/balances.js
const ethers = require('ethers');

const providerRPC = {
  development: {
    name: 'moonbeam-development',
    rpc: 'http://localhost:9944',
    chainId: 1281,
  },
  moonbase: {
    name: 'moonbase-alpha',
    rpc: 'https://rpc.api.moonbase.moonbeam.network',
    chainId: 1287,
  },
};
const provider = new ethers.JsonRpcProvider(providerRPC.development.rpc, {
  chainId: providerRPC.development.chainId,
  name: providerRPC.development.name,
}); // Change to correct network

const addressFrom = 'ADDRESS-FROM-HERE';
const addressTo = 'ADDRESS-TO-HERE';

const balances = async () => {
  const balanceFrom = ethers.formatEther(await provider.getBalance(addressFrom));

  const balanceTo = ethers.formatEther(await provider.getBalance(addressTo));

  console.log(`The balance of ${addressFrom} is: ${balanceFrom} ETH`);
  console.log(`The balance of ${addressTo} is: ${balanceTo} ETH`);
};

balances();

File: .snippets/code/web3py-tx/balances.py
# 1. Add import
from web3 import Web3

# 1. Add the Web3 provider logic here:
provider_rpc = {
    'development': 'http://localhost:9944',
    'alphanet': 'https://rpc.api.moonbase.moonbeam.network',
}
web3 = Web3(Web3.HTTPProvider(provider_rpc['development']))  # Change to correct network

# 2. Create address variables
address_from = 'ADDRESS_FROM_HERE'
address_to = 'ADDRESS_TO_HERE'

# 3. Fetch balance data
balance_from = web3.fromWei(web3.eth.getBalance(address_from), 'ether')
balance_to = web3.fromWei(web3.eth.getBalance(address_to), 'ether')

print(f'The balance of { address_from } is: { balance_from } ETH')
print(f'The balance of { address_to } is: { balance_to } ETH')